plugins/telemetry/telemetry.h

```diff
@@ -6,48 +6,90 @@ namespace dronecore {
 
 class TelemetryImpl;
 
+/**
+ * The Telemetry class allows to get all kinds of telemetry infos from a device.
+ */
 class Telemetry
 {
 public:
+    /**
+     * Constructor for Telemetry called internally.
+     */
     explicit Telemetry(TelemetryImpl *impl);
+
+    /**
+     * Destructor for Telemetry called internally.
+     */
     ~Telemetry();
 
+    /**
+     * Position type.
+     */
     struct Position {
-        double latitude_deg;
-        double longitude_deg;
-        float absolute_altitude_m;
-        float relative_altitude_m;
+        double latitude_deg; /**< latitude from -90 to +90 in degrees */
+        double longitude_deg; /**< longitude from -180 to 180 in degrees */
+        float absolute_altitude_m; /**< altitude AMSL (above mean sea level in meters */
+        float relative_altitude_m; /**< altitude relative to takeoff altitude in meters */
     };
 
+    /**
+     * Quaternion type
```

Comment:
This means nothing to me. Can we have a link to more information?

Reply:
Sure, can link to wikipedia.